app/js/components/searchHistory/searchHistoryComponent.jsx

```diff
@@ -70,7 +70,7 @@ class SearchHistoryComponent extends Component {
                                     {
                                         this.props.history.map((eachResult, index) =>
                                             <tr key={shortId.generate()}>
-                                                <td>{this.props.history.length - index}</td>
+                                                <td>{index + 1}</td>
```

Comment:
Uhmmnnn I intentionally did this because of composition search, the number has to correspond with the index positioning. I wanted to try as much as possible to maintain the fact that the search with serial number 1 is the first search not the latest search, cos the latest search will constantly change